modules/ossm-config-individual-control-plane-infrastructure-node-console.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-config-individual-control-plane-infrastructure-node-console_{context}"]
+= Configuring individual control plane components to run on infrastructure nodes using the web console
+
+You can configure individual {SMProductShortName} control plane components to run on infrastructure nodes using the CLI. This task should only be performed if individual {SMProductShortName} control plane components, such as Istiod, the Ingress Gateway, and the Egress Gateway, run on infrastructure nodes. 
+
+If the control plane runs on a worker node, skip this task.
+
+.Prerequisites
+
+* The {SMProductName} Operator must be installed.
+* An account with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Create a project named `istio-system`.
++
+.. Navigate to *Home* -> *Projects*.
++
+.. Click *Create Project*.
++
+.. In the *Name* field, enter `istio-system`. The `ServiceMeshControlPlane` resource must be installed in a project that is separate from your microservices and Operators.
++
+These steps use `istio-system` as an example, but you can deploy your {SMProductShortName} control plane in any project as long as it is separate from the project that contains your services.
++
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Click the {SMProductName} Operator, then click *Istio Service Mesh Control Plane*.
+
+. Click the name of the control plane resource. For example, `basic`.
```

Comment:
Same as above: the SMCP won't exist here, so the step explain creating the SMCP instead of editing it.

Reply:
Done, removed steps to create the SMCP.